src/diffusers/modeling_flax_utils.py

```diff
@@ -339,6 +340,7 @@ def from_pretrained(
                     f"Error no file named {FLAX_WEIGHTS_NAME} or {WEIGHTS_NAME} found in directory "
                     f"{pretrained_path_with_subfolder}."
                 )
+            send_telemetry({"model_path": "local", "framework": "flax"}, name="diffusers_from_pretrained")
```

Comment:
We will need to know which model class is used. Could you maybe also add a `{"model_class": cls.__name__` to it?